public/index.html

```diff
@@ -39,72 +36,45 @@
+            <a-sphere scale="0.1 0.1 0.1"></a-sphere>
+            <a-entity id="head" camera="userHeight: 1.6" look-controls networked="template:#head-template;showLocalTemplate:false;"></a-entity>
 
-        <a-entity
-            id="nametag"
-            networked="template:#nametag-template;showLocalTemplate:false;"></a-entity>
+            <a-entity id="nametag" networked="template:#nametag-template;showLocalTemplate:false;"></a-entity>
 
-        <a-entity
-            id="left-hand"
-            hand-controls="left"
-
-            axis-dpad="centerZone: 1"
-            teleport-controls="cameraRig: #player-rig; teleportOrigin: #head; button: action_teleport_"
-            networked="template:#hand-template;showLocalTemplate:false;"
-        >
-            <a-entity id="watch" gltf-model="assets/hud/watch.gltf" position="0 0.0015 0.147" rotation="3.5 0 0">
-                <a-circle
-                    mute-state-indicator
-                    scale-audio-feedback="analyserSrc: #head; minScale: 0.035; maxScale: 0.08;"
-                    position="0 0.023 0"
-                    rotation="-90 0 0"
-                    scale="0.04 0.04 0.04"
-                    material="color:#d8eece;shader:flat">
-                </a-circle>
+            <a-entity id="left-hand" hand-controls="left" axis-dpad="centerZone: 1" teleport-controls="cameraRig: #player-rig; teleportOrigin: #head; button: action_teleport_"
+                networked="template:#hand-template;showLocalTemplate:false;">
+                <a-entity id="watch" gltf-model="assets/hud/watch.gltf" position="0 0.0015 0.147" rotation="3.5 0 0">
+                    <a-circle mute-state-indicator scale-audio-feedback="analyserSrc: #head; minScale: 0.035; maxScale: 0.08;" position="0 0.023 0"
+                        rotation="-90 0 0" scale="0.04 0.04 0.04" material="color:#d8eece;shader:flat">
+                    </a-circle>
+                </a-entity>
             </a-entity>
-        </a-entity>
 
-        <a-entity
-            id="right-hand"
-
-            hand-controls="right"
-
-            axis-dpad
-            teleport-controls="cameraRig: #player-rig; teleportOrigin: #head; hitEntity: #telepor-indicator; button: action_teleport_;"
-            networked="template:#hand-template;showLocalTemplate:false;"></a-entity>
-    </a-entity>
+            <a-entity id="right-hand" hand-controls="right" axis-dpad teleport-controls="cameraRig: #player-rig; teleportOrigin: #head; hitEntity: #telepor-indicator; button: action_teleport_;"
+                networked="template:#hand-template;showLocalTemplate:false;"></a-entity>
+        </a-entity>
 
-    <a-entity class="head" gltf-model="#rock-island" position="0 0 0">
-    <a-sky color="#DDFFD9"></a-sky>
-  </a-scene>
-  <script>
-   document.querySelector('a-scene').addEventListener('loaded', onSceneLoad)
-  </script>
+        <a-entity class="head" gltf-model="#rock-island" position="0 0 0">
```

Comment:
Ah actually looks like an unclosed tag here, wonder if that's why indenting got all screwey